portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getServerStats()">
```

Comment:
Since we're not doing the examples anymore, getServerStats() is no longer a good name. Let's rename it getComments()/getMessages() here and in the JS file?